.storybook/preview-head.html

```diff
@@ -1,4 +1,6 @@
-<link rel="stylesheet" href="%STORYBOOK_PROXIMA_NOVA_LINK%">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&display=swap" rel="stylesheet">
```

Comment:
I see that figma is using 400 (regular), 600(semi-bold) and 700(bold). https://www.figma.com/file/vnzYfrwIISRYkr2iYE5TcD/DS-Atoms?node-id=27299%3A41324